### DISCUSSION TITLE
GSSAPI and SPNEGO

### DISCUSSION THREAD

**Discussion by aquaherd** [qutebrowser/qutebrowser#6229] [Q&A] [answered]
If I compare edge and qutebrowser side-by-side in a corporate Windows environment, I see that qutebrowser asks for intranet website credentials while edge never does this. I guess that kerberos authentication with gssapi / spnego is behind this feature. 

Can I somehow replicate this comfort with cutebrowser? 

Perhaps like in `:set qt.environ QT_SOME_KERBEROS_ALLOW_REALMS="somecorp.tld, someothercorp.tld"` ?

**Comment by The-Compiler** [comment, marked as answer]
#4595 talks about an `auth-server-whitelist` argument you can add to `qt.args` - though I've never used it myself.

**Reply by webknjaz** [reply]
Additional detail: this works as long as `qtwebengine` is compiled with `kerberos` support and is a no-op otherwise. Ref https://github.com/qutebrowser/qutebrowser/issues/4595#issuecomment-2369165892.
